src/content/changes/managed-apps/kong-app/v4.0.0.md

```diff
@@ -17,7 +17,7 @@ title: kong-app release v4.0.0
 This is a major release because it changes the default behaviour of the Chart.
 - For Kong Enterprise deployment the flag `.Values.image.repository` and `.Values.image.tag` can now be omitted.
 - For Kong OSS deployment the flag `.Values.image.repository` and `.Values.image.tag` now have to be set to the respective OSS values.
-For further instructions please refer to the [README.md](https://github.com/giantswarm/kong-app/blob/main/README.md).
+For further instructions please refer to the [README.md](./README.md).
```

Comment:
@pipo02mix At best we would fix relative links when copying these Markdown files

Reply:
We definitely can't fix it here manually 🤷

Reply:
you mean to fix it in the automation? it is a good idea, yeah

Reply:
I will address this in incoming PR